packages/salesforcedx-vscode-core/src/commands/stopApexDebugLogging.ts

```diff
@@ -15,11 +15,20 @@ const command = 'stop_apex_debug_logging';
 export const turnOffLogging = async (): Promise<void> => {
   handleStartCommand(command);
 
-  if (developerLogTraceFlag.isActive()) {
+  const connection = await WorkspaceContext.getInstance().getConnection();
+
+  // Check if a DebugLevel with DeveloperName 'ReplayDebuggerLevels' already exists
+  const replayDebuggerLevels = await connection.tooling.query(
+    "SELECT id, logtype, startdate, expirationdate, debuglevelid, debuglevel.apexcode, debuglevel.visualforce FROM TraceFlag WHERE logtype='DEVELOPER_LOG'"
```

Comment:
@daphne-sfdc no need to return all of these named fields if all that is needed is "id"
``` suggestion
    "SELECT id FROM TraceFlag WHERE logtype='DEVELOPER_LOG'"
```